0000-named-tracers.md

```diff
@@ -0,0 +1,67 @@
+# Named Tracers 
+
+_Creating tracers using a factory mechanism and naming those tracers in accordance with the (logical) components being traced._
```

Comment:
"in accordance with the (logical) components being traced" -> "in accordance with the library that provides the instrumentation for a traced component."

Reply:
Addressed in follow-up commit.

Reply:
Addressed in follow-up commit.